release-calendar/src/client/components/App.tsx

```diff
@@ -13,10 +13,41 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import * as React from 'react';
+import {Calendar} from './Calendar';
+//TODO: remove DATA and instead populate with data from ./test/development-data.ts
+import {DATA} from '../models/data';
+import {EventSourceInput} from '@fullcalendar/core/structs/event-source';
 import {Header} from './Header';
+import {getEvents} from '../models/release-event';
+
+interface AppState {
+  events: EventSourceInput[];
+}
+
+export class App extends React.Component<{}, AppState> {
+  constructor(props: unknown) {
+    super(props);
+    this.state = {
+      events: [],
+    };
+  }
+
+  //TODO: change fetch API call to service
+  componentDidMount(): void {
+    Promise.resolve(getEvents(DATA)).then(result =>
+      this.setState({events: result}),
+    );
+  }
 
-export const App = (): JSX.Element => {
-  return <Header title='AMP Release Calendar' />;
-};
+  render(): JSX.Element {
+    return (
+      <div className='AMP-Release-Calendar'>
```

Comment:
Where is this class name used? Also, css class names are typically all undercase and preferably short.

Reply:
same as above